src/components/DelegatesTable/DelegatesTable.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <v-table :class="classes.root">
+    <delegates-table-head />
+    <tbody>
+      <template v-if="searchDelegates.length > 0">
+        <delegates-table-item
+          v-for="(delegate) in searchDelegates"
+          :key="delegate.username"
+          :delegate="delegate"
+          :details-expanded="expandedDelegateUsername === delegate.username"
+          @update:details-expanded="state => { updateExpandedDelegateUsername(delegate.username)(state) }"
+        />
+      </template>
+      <delegates-loader v-else-if="delegates.length === 0" :waiting-for-confirmation="waitingForConfirmation" />
```

Comment:
split the attributes to several lines, at first I thought the `=== 0" : waiting...` is a ternary operator